.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,10 +3,10 @@
 ## About your game
 
 What is your game about? 
-> Example: Pushing boxes to the goal. (from [Sokoban Plus](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/sokoban_plus.js))
+<!-- Example: Pushing boxes to the goal. (from [Sokoban Plus](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/sokoban_plus.js)) -->
 
 How to play your game?
-> Example  : Press WASD to move, J to restart and K to toggle trails, Get A boxes (cyan) to A goals (green), Get B boxes (magenta) to B goals (red), Get normal boxes (gray) to either goal. (from [Sokoban plus](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/sokoban_plus.js))
+<!-- Example  : Press WASD to move, J to restart and K to toggle trails, Get A boxes (cyan) to A goals (green), Get B boxes (magenta) to B goals (red), Get normal boxes (gray) to either goal. (from [Sokoban plus](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/sokoban_plus.js)) -->
 
 ## Code
 Check off the items that are true.

```

Comment:
```suggestion
Check off the items that are true. <-- Put a `x` in the  `[ ]`. Example: `[x] The game was made...` -->
```
This would be helpful to those who have trouble checking the boxes.